lib/ui/common/app_colors.dart

```diff
@@ -7,3 +7,12 @@ const Color kcMediumGrey = Color(0xFF474A54);
 const Color kcLightGrey = Color.fromARGB(255, 187, 187, 187);
 const Color kcVeryLightGrey = Color(0xFFE3E3E3);
 const Color kcBackgroundColor = kcDarkGreyColor;
+const Color dullGreen = Color(0xff71A968);
+const Color appBarYellow = Color(0xffDBA31D);
+const Color darkDirtyGreen = Color(0xff525926);
+const Color veryLightGrey = Color(0xffF8F8F6);
+const Color blue = Color(0xff144B69);
+const Color green = Color(0xff1A8F54);
+const Color grey = Color(0xff9BB4A0);
+const Color textColor = Color(0xff333930);
+const Color darkBlue = Color(0xff1F1A44);
```

Comment:
I will like us to have our colors in this format:

```
const Color primary = //the app primary color
const Color primaryDark = //the app primary color
...
// If there is secondary color or general colors, we can now name them
const Color gray = 
const Color yellow =
const Color green =
```
Then remove the default colors set by Stacked, since we are not using it.